sdk/appconfiguration/app-configuration/README.md

```diff
@@ -1,28 +1,91 @@
-# Azure App Configuration client library for JS
+# App Configuration client library for TypeScript/JavaScript
 
-Azure App Configuration is a managed service that helps developers
-centralize their application configurations, simply and securely.
+[Azure App Configuration][appconfig_concepts] is a managed service that helps developers centralize their application and feature settings simply and securely.
 
-This client library lets you create, retrieve, update, and delete 
-settings within Azure App Configuration.
+Use the client library for App Configuration to:
+* Create flexible key representations and mappings
+* Tag keys with labels
+* Replay settings from any point in time
 
-* [Azure App Configuration documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
-* [NPM](https://www.npmjs.com/package/@azure/app-configuration)
-* [Source](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/)
-* [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/appconfiguration/app-configuration/samples)
+[Source](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/) | [NPM](https://www.npmjs.com/package/@azure/app-configuration) | [API Reference documentation](https://azure.github.io/azure-sdk-for-js/app-configuration/) | [Samples][samples]
```

Comment:
The other readmes also list a link to the product documentation on docs.microsoft.com.

For app config: 

`[Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)`

Reply:
Oh and also you should use full github paths instead of relative paths in the links to the samples folder on this page, since the readme will also show up on NPM and other places and that will make sure that it always links back to github for samples.

Reply:
Ah, you're talking about the <!-- links thing at the bottom. I've inlined all the links now, no more fanciness. :)